sdk/translation/Azure.AI.Translation.Text/src/Azure.AI.Translation.Text.csproj

```diff
@@ -2,9 +2,7 @@
   <PropertyGroup>
     <Description>This is the Text Translation client library for developing .NET applications with rich experience.</Description>
     <AssemblyTitle>Azure SDK Code Generation Text Translation for Azure Data Plane</AssemblyTitle>
-    <Version>1.1.0-beta.1</Version>
-    <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>1.0.0</ApiCompatVersion>
```

Comment:
Do not remove the API Compat version.   Revert this change, please.

Reply:
I think someone removed it by accident previously. I will add it back.
What ApiCompatVersion should I use?